README.md

```diff
@@ -1,18 +1,20 @@
-## Ops Toolbelt
+# Ops Toolbelt
 
 [![REUSE status](https://api.reuse.software/badge/github.com/gardener/ops-toolbelt)](https://api.reuse.software/info/github.com/gardener/ops-toolbelt)
 
-### What is the ops-toolbelt
-The `ops-toolbelt` aims to be a standard operator container image with pre-installed useful tools for troubleshooting issues on gardener landscapes. The `ops-toolbelt` images can be used by the Gardener Dashboard web consoles functionality to log into the garden, seed, or shoot clusters.
+## What is the ops-toolbelt?
 
-The pods created with this image can be both general pods and node-bound pods (behaving as if being on the node directly)
-Starting a pod with the ops-toolbel timage requires a running Kubelet and healthy control plane, a working VPN connection, and sufficient capacity on the node.
+The `ops-toolbelt` aims to be a standard container image with pre-installed useful tools for troubleshooting issues on gardener landscapes for human operators. The `ops-toolbelt` images can be used by the Gardener Dashboard web consoles functionality to log into the garden, seed, or shoot clusters.
```

Comment:
> The `ops-toolbelt` images can be used by the Gardener Dashboard web consoles functionality to log into the garden, seed, or shoot clusters.

not sure we want to mention it here

Reply:
Yeah, I wasn't sure either but I didn't want to remove things which I wasn't sure about! Thanks for the feedback.

Reply:
Addressed in [0102cad](https://github.com/gardener/ops-toolbelt/pull/142/commits/0102cad035b1842c8cb187e8f670ff6aa5a7cf45).